src/main/java/org/jenkinsci/remoting/engine/JnlpConnectionState.java

```diff
@@ -23,6 +23,7 @@
  */
 package org.jenkinsci.remoting.engine;
 
```

Comment:
Is something outdated or incomplete in this file? I'm not seeing that any of the changes here are actually needed.

Reply:
You need to look at the upstream PR.

Reply:
I'm only part way through that one. Sounds good.